cmd/main.go

```diff
@@ -25,6 +27,7 @@ func main() {
 
 	configPath := flag.String("cp", "../config", "Path to config file")
 	configName := flag.String("cn", "viper.config", "Name of config file")
+	servicePort := flag.String("p", ":8000", "Service port")
```

Comment:
Value `:8000` is not a port, it is format: listen on port 8000 on any interface

Reply:
fixed